lib/rubocop/cop/mixin/hash_shorthand_syntax.rb

```diff
@@ -7,7 +7,7 @@ module HashShorthandSyntax
       OMIT_HASH_VALUE_MSG = 'Omit the hash value.'
       EXPLICIT_HASH_VALUE_MSG = 'Explicit the hash value.'
 
-      def on_pair(node)
+      def on_pair(node) # rubocop:todo Metrics/MethodLength
```

Comment:
It could be changed as follows instead of `rubocop:todo Metrics/MethodLength` comment:

```diff
diff --git a/lib/rubocop/cop/mixin/hash_shorthand_syntax.rb b/lib/rubocop/cop/mixin/hash_shorthand_syntax.rb
index 106774c05..25d2285cd 100644
--- a/lib/rubocop/cop/mixin/hash_shorthand_syntax.rb
+++ b/lib/rubocop/cop/mixin/hash_shorthand_syntax.rb
@@ -7,7 +7,7 @@ module RuboCop
       OMIT_HASH_VALUE_MSG = 'Omit the hash value.'
       EXPLICIT_HASH_VALUE_MSG = 'Explicit the hash value.'

-      def on_pair(node) # rubocop:todo Metrics/MethodLength
+      def on_pair(node)
         return if ignore_hash_shorthand_syntax?(node)

         hash_key_source = node.key.source
@@ -25,13 +25,17 @@ module RuboCop
           replacement = "#{hash_key_source}: #{hash_key_source}"
         end

+        register_offense(node, message, replacement)
+      end
+
+      private
+
+      def register_offense(node, message, replacement)
         add_offense(node.value, message: message) do |corrector|
           corrector.replace(node, replacement)
         end
       end

-      private
-
       def ignore_hash_shorthand_syntax?(pair_node)
         target_ruby_version <= 3.0 || enforced_shorthand_syntax == 'either' ||
           !pair_node.parent.hash_type?
```

Reply:
I've updated the branch. Thank you!